packages/boilerplate/src/router/index.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable */
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import * as Rudy from '@respond-framework/rudy'
+
+const routesMap = {}
+
+class Router extends Component {
+  constructor(props) {
+    super(props);
+
+    const childs = Array.isArray(props.children) ? props.children : [props.children]
+
+    const routeKey = childs[0].props.component().chunkName()
+    const routes = {
```

Comment:
react-router does not require that `<Route />` is a direct child of `<Router />`: https://reacttraining.com/react-router/core/api/Route

Do you think its necessary to support that? It could probably be implemented using a react context prop indicating what the ID of the closest parent `<Route />` component is (then the `<Route />`s could dispatch actions on mount/unmount adding/removing their subroutes.

Reply:
having built in support for child/nested routes would be useful here, since otherwise the route in routeMap generated for each `<Route />` would have to do all the work to compose any parent `<Route />`s it was rendered under.